qubes_menu/appmenu.py

```diff
@@ -264,23 +264,23 @@ def exit_app(self):
                 task.cancel()
 
 
+from .tests.mock_app import new_mock_qapp
```

Comment:
Mocking should not happen in the application code itself, but exclusively in the tests.

Reply:
I don't want to run the app in dom0, and I need a way to run the app.

Reply:
Then, create a file in `tests` to run it with `qubesadmin.Qubes()` patched. Production code should not import tests under any circumstances.